<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>savepackagewaitpage</class>
 <widget class="QWidget" name="savepackagewaitpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>80</number>
     </property>
     <property name="topMargin">
      <number>165</number>
     </property>
     <property name="bottomMargin">
      <number>70</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>timeLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>200</number>
       </property>
       <property name="rightMargin">
        <number>200</number>
       </property>
       <property name="bottomMargin">
        <number>60</number>
       </property>
       <item>
        <widget class="QLabel" name="door">
         <property name="text">
          <string>door</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="doorCabinetId">
         <property name="text">
          <string>doorCabinetId</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="doorCabinet">
         <property name="text">
          <string>doorCabinet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="doorCellId">
         <property name="text">
          <string>doorCellId</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="doorCell">
         <property name="text">
          <string>doorCell</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QPushButton" name="okBtn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
